collaboration_cafes/index.md

```diff
@@ -0,0 +1,27 @@
+
+# SATRE Collaboration Cafes Schedule
+
+
+## Schedule
+
+| Date | Time | Chair | HackMD Notes | Focus / Theme |
+| ---- | ---- | ----- | ------------ | ------------- |
+| 04 April | 15:00 - 16:00 | Hari Sood (hsood@turing.ac.uk) | [Link to notes](./20230404-general.md) | |
+| 20 April | 15:00 - 16:00 | Hari Sood (hsood@turing.ac.uk) | [Link to notes](./20230419-general.md) | |
+| 02 May | 15:00 - 16:00 | Simon Li (s.p.li@dundee.ac.uk) | [Link to notes](./20230502-general.md) | |
+| 18 May | 15:00 - 16:00 | Arron Lacey (alacey@turing.ac.uk) | [Link to notes](./20230518-survey_share_out.md) | Survey share-out |
+| 06 June | 15:00 - 16:00 | Hari Sood (hsood@turing.ac.uk) |[Link to notes](./20230606-information_goevrnance.md) | Information Governance |
```

Comment:
Typo in `information_goevrnance.md` (and in the original filename), otherwise LGTM!

Reply:
Sorted